presto-docs/src/main/sphinx/functions/json.rst

```diff
@@ -66,19 +66,21 @@ JSON Functions
 
         SELECT json_array_contains('[1, 2, 3]', 2);
 
-.. function:: json_array_get(json_array, index) -> varchar
+.. function:: json_array_get(json_array, index) -> json
+
+   .. warning:: The semantics of this function is broken. We recommend against using it.
```

Comment:
Let's change this to
```
   .. warning::

       The semantics of this function are broken. If the extracted element
       is a string, it will be converted into an invalid ``JSON`` value that
       is not properly quoted (the value will not be surrounded by quotes
       and any interior quotes will not be escaped).

       We recommend against using this function. It cannot be fixed without
       impacting existing usages and may be removed in a future release.